### PR TITLE
Aaa

### PR DESCRIPTION
**Pull request by Antonieta13**
AAA pattern

### CODE REVIEW

File: cypress/integration/dresses-list.spec.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    let menuContentPage: MenuContentPage;
+    let dressesListPage: DressesListPage;
+
+    before(() => {
+        menuContentPage = new MenuContentPage();
+        dressesListPage = new DressesListPage();
+    })
+
+    it("show the available dresses", () => {
+
```
Comment (holgiosalos) [inline cypress/integration/dresses-list.spec.ts]: remove this blank line

File: cypress/integration/dresses-list.spec.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        const DressesNames = ["Printed Dress", "Printed Dress", "Printed Summer Dress", "Printed Summer Dress", "Printed Chiffon Dress"];
+
+        menuContentPage.visitMenuContentPage()
+        menuContentPage.goToDressesMenu()
+        dressesListPage.validateItemsNumber(5)
```
Comment (holgiosalos) [inline cypress/integration/dresses-list.spec.ts:23]: Add a blank line here, to separathe the act (visitMenuContentPage and goToDressesMenu) from the assert (the validate methods)

File: cypress/page/dresses-list.page.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        cy.get(this.dressName).each((item, index) => {
+            cy.wrap(item).should('contain.text', names[index])
+        })
+    }
+
+}
+
+export { DressesListPage }
+
+//DressesNames
```
Comment (holgiosalos) [inline cypress/page/dresses-list.page.ts]: remove this two lines

File: cypress/page/login.page.ts [hunk 4]
```diff
@@ -18,7 +18,8 @@ class LoginPage {
         cy.get(this.passInput).type(pass)
     }
 
-    public login(): void {
+    public login
```
Comment (holgiosalos) [inline cypress/page/login.page.ts]: leave the `public login(): void {` on the same line

File: cypress/integration/dresses-list.spec.ts (hunk 1 above)
Comment (Antonieta13) [reply cypress/integration/dresses-list.spec.ts]: done

File: cypress/integration/dresses-list.spec.ts (hunk 2 above)
Comment (Antonieta13) [reply cypress/integration/dresses-list.spec.ts:23]: Done

File: cypress/page/dresses-list.page.ts (hunk 3 above)
Comment (Antonieta13) [reply cypress/page/dresses-list.page.ts]: Done

File: cypress/page/login.page.ts (hunk 4 above)
Comment (Antonieta13) [reply cypress/page/login.page.ts]: Done

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**